src/lib/ui/components/File/File.svelte

```diff
@@ -45,8 +45,11 @@
           ? ([...files, ...filesToUpload] as File[])
           : filesToUpload[0];
 
-      if (fileToUpload instanceof File) {
-        const sizeKB = fileToUpload.size / 1024;
+      if (fileToUpload instanceof File || Array.isArray(fileToUpload)) {
+        const sizeKB =
+          (fileToUpload instanceof File
+            ? fileToUpload.size
+            : fileToUpload.reduce((acc, file) => acc + file.size, 0)) / 1024;
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/no-shadow](https://typescript-eslint.io/rules/no-shadow)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'file' is already declared in the upper scope on line 30 column 6.